packages/teko/src/Teko_ConfigDefs.hpp

```diff
@@ -54,7 +54,13 @@
 namespace Teko {
 
 typedef double ST;
-typedef int LO;
+using LO = Tpetra::Map<>::local_ordinal_type;
```

Comment:
I think this change is fine, especially since we plan to land the big "remove deprecated code" PR #6388 tomorrow.  That PR would make it so that Tpetra only ever has one `GO` type enabled.  Thus, it would be correct for Teko to use Tpetra's default types.

Reply:
Great.  The removal of the deprecated StaticGraph code PR #6388 today is why I was hoping to get this PR in sooner rather than later....